gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineTesting.java

```diff
@@ -140,8 +142,17 @@ private static void enableJunit5ForTestTask(Test task) {
         // Computes the desired parallelism based on the number of available processors/cores
         task.systemProperty("junit.jupiter.execution.parallel.config.strategy", "dynamic");
 
-        // provide some stdout feedback when tests fail
-        task.testLogging(testLogging -> testLogging.events("failed"));
+        // provide some stdout feedback when tests fail when running on CI and locally
+        task.getTestLogging().getEvents().add(TestLogEvent.FAILED);
+
+        // Only on CI, print out more detailed test information to avoid hitting the circleci 10 min deadline if
+        // there are lots of tests. Don't do this locally to avoid spamming massive amount of info for people running
+        // unit tests through the command line
+        if ("true".equals(System.getenv("CI"))) {
+            task.getTestLogging()
```

Comment:
@CRogers can we try to not apply this to the `./gradlew test` tasks which are normally just unit-tests??

Reply:
Yep, I have a commit to do this actually, I was just distracted by testing it on c-v-s that had the silently not running tests problem :p

Reply:
Pushed